core-services/prow/02_config/openshift/coredns-ocp-dnsnameresolver/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/coredns-ocp-dnsnameresolver
+  require_self_approval: false
```

Comment:
This setting reduces the number of people required to make the PR pass from 2 to 1 if the creator is from the OWNERS list. I think it's better to have an explicit approval and LGTM labels from 2 different reviews, especially for a code shared between teams.